src/db/opt.rs

```diff
@@ -0,0 +1,78 @@
+use super::*;
+
+#[derive(Clone)]
+pub struct AgateOptions {
+    pub path: PathBuf,
+    pub value_dir: PathBuf,
+    // TODO: docs
+    pub in_memory: bool,
+    pub sync_writes: bool,
+
+    // Memtable options
+    pub mem_table_size: u64,
+    pub base_table_size: u64,
+    pub base_level_size: u64,
+    pub level_size_multiplier: usize,
+    pub table_size_multiplier: usize,
```

Comment:
What is `table_size_multiplier`?

Reply:
This option is defined at https://github.com/dgraph-io/badger/blob/master/options.go#L58 , which seems to be used when calculating target file size of a LSM level.

Reply:
Turns out I'm referencing an outdated version. 🤣

Reply:
I have forked badger@45bca18 at https://github.com/skyzh/badger . Seems that badger is under constant change, with new things added and some logic changed. We could always use my fork as reference implementation.